services/users-api/lambdas/deleteAttachment.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+import to from 'await-to-js';
+import { throwError, BadRequestError } from '@helsingborg-stad/npm-api-error-handling';
+import S3 from '../../../libs/S3';
+import * as response from '../../../libs/response';
+import { decodeToken } from '../../../libs/token';
+
+const BUCKET_NAME = process.env.BUCKET_NAME;
+
+export async function main(event) {
+  const { filename } = event.pathParameters;
+  if (!filename) {
+    return response.failure(new BadRequestError('Missing filename in path query string'));
+  }
+
+  const decodedToken = decodeToken(event);
+  const { personalNumber } = decodedToken;
+
+  const [getFilesError, userS3Files] = await to(getFilesFromUserS3Bucket(personalNumber));
+  if (getFilesError) {
+    return response.failure(getFilesError);
+  }
+
+  const fileToDelete = `${personalNumber}/${filename}`;
```

Comment:
I don't mind this naming, i think it works. But for some reason i feel that it might be misleading, since it's the filePath or fileName rather that is declared and not the file in bytes itself.

Reply:
Maybe fileKey or filePath would be better?

Reply:
Corrected